Formula/pgcli.rb

```diff
@@ -67,6 +69,9 @@ class Pgcli < Formula
   end
 
   def install
+    # Append Homebrew's bin directory to the PATH so Psycopg can find the pg_config binary.
+    ENV.append_path "PATH", "#{HOMEBREW_PREFIX}/bin"
```

Comment:
Will this append to the path every time we install or upgrade even if `#{HOMEBREW_PREFIX}/bin` is already in the path?